source/docs/v3/integrations/sail.blade.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Laravel Sail integration
+extends: _layouts.documentation
+section: content
+---
+
+# Laravel Sail {#sail}
+
+## Central domains
+
+If you're using Laravel Sail, no changes are needed, default values are good to go:
+
+```php
+// config/tenancy.php
+
+'central_domains' => [
+    '127.0.0.1',
+    'localhost',
+],
+```
+
+## Creating tenants
+
+Before creating any tenants you have to ensure that the `sail` user has the necessary permissions.
+
+To do so either login to the mysql shell or via any client using root user credentials and run the following queries:
+
+```bash
```

Comment:
Both of these code blocks should be `sql`, not `bash`

Reply:
Yes I know, I intentionally used `bash` because of styling issue.

![image](https://user-images.githubusercontent.com/70144052/193395395-792d6534-0a96-4a6f-bc3a-f8beb1c2ce76.png)

The first one is `bash`, and the second one is `sql`. Can someone look into the CSS part?

Reply:
I see, that's probably because we don't have sql in the languages for our code highlighter. `bash` is fine then

Reply:
Seems like you change this to `sql` but didn't update the styles? So now it looks like on the screenshot above https://6345ccd6c6181c000829be1d--stancl-tenancy.netlify.app/docs/v3/integrations/sail/

Reply:
Okay, I must have not committed the assets. I should run `npm run prod`, and then commit right?

Reply:
Fixed